src/borg/testsuite/benchmark.py

```diff
@@ -148,7 +148,7 @@ def do_gets(keys=keys):
 
 def test_chunk_indexer_getitem_c(benchmark):
     import itertools
-    max_key = 2**20
+    max_key = int(1287917 * 0.93 - 10)
```

Comment:
this is only "matching" the robin hood hashtable implemention, to compare with the current one, we would need the other load factor here. maybe use load_factor = 0.93 here or even import it from hashindex.pyx.